src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -236,12 +236,21 @@ private IList<string> GetResourceNamesForCulture(CultureInfo culture)
             {
                 var names = new List<string>();
```

Comment:
A using block counts as a different scope so it would be inaccessible to the return if i put it in there, so I have to move the return up into it too,